src/System.Management.Automation/engine/Utils.cs

```diff
@@ -895,113 +895,49 @@ internal static bool IsAdministrator()
 #endif
         }
 
-        internal static bool NativeItemExists(string path)
+        internal static bool ItemExists(string path)
         {
-            bool unusedIsDirectory;
-            Exception unusedException;
-
-            return NativeItemExists(path, out unusedIsDirectory, out unusedException);
+            bool result = false;
+            try
+            {
+                result =  ItemExists(path, out bool unused);
+            }
+            catch
+            {
+            }
+            return false;
         }
 
-        // This is done through P/Invoke since File.Exists and Directory.Exists pay 13% performance degradation
```

Comment:
Are we sure we no longer have a perf penalty with this change?

Reply:
We have still a perf penalty (on windows because normalization(!) - one extra kernel call in .Net code) but it is tracked in CoreFX https://github.com/dotnet/corefx/issues/19717
We have already mentioned that we do up to 7 the normalizations of the same path per cmdlet call. So perf penalty is more in PowerShell code. I'm trying to find how we can reduce the path normalization perf penalty. 

Also please see https://github.com/dotnet/corefx/issues/27049#issuecomment-392826352